src/main/java/com/minecolonies/coremod/entity/pathfinding/AbstractPathJob.java

```diff
@@ -268,7 +268,7 @@ protected static double computeCost(Node parent, @NotNull BlockPos dPos, boolean
             cost *= 1.1D;
         }
 
-        if(onPath)
+        if (onPath)
         {
             cost *= 0.75D;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.75D to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)